subworkflows/nf-core/variantcalling/controlfreec/main.nf

```diff
@@ -0,0 +1,108 @@
+include { CAT_CAT as CAT_MPILEUP_NORMAL                          } from '../../../../modules/nf-core/modules/cat/cat/main.nf'
+include { CAT_CAT as CAT_MPILEUP_TUMOR                           } from '../../../../modules/nf-core/modules/cat/cat/main.nf'
+include { CONTROLFREEC_FREEC as FREEC                            } from '../../../../modules/nf-core/modules/controlfreec/freec/main'
+include { CONTROLFREEC_ASSESSSIGNIFICANCE as ASSESS_SIGNIFICANCE } from '../../../../modules/nf-core/modules/controlfreec/assesssignificance/main'
+include { CONTROLFREEC_FREEC2BED as FREEC2BED                    } from '../../../../modules/nf-core/modules/controlfreec/freec2bed/main'
+include { CONTROLFREEC_FREEC2CIRCOS as FREEC2CIRCOS              } from '../../../../modules/nf-core/modules/controlfreec/freec2circos/main'
+include { CONTROLFREEC_MAKEGRAPH as MAKEGRAPH                    } from '../../../../modules/nf-core/modules/controlfreec/makegraph/main'
+include { SAMTOOLS_MPILEUP as MPILEUP_NORMAL                     } from '../../../../modules/nf-core/modules/samtools/mpileup/main'
+include { SAMTOOLS_MPILEUP as MPILEUP_TUMOR                      } from '../../../../modules/nf-core/modules/samtools/mpileup/main'
+
+workflow RUN_CONTROLFREEC {
```

Comment:
@maxulysse I wanted to ask your opinion on this. In theory it would work beautifully to have one Controlfreec workflow for tumor and t/n. In the first case we just skip all the "normal" steps. Now once again we have the problem that with `when` the processes still get printed to the terminal. Surrounding the individual modules with `if` we already discussed in a different context and is generally undesirable. But basically c&p the workflow and removing 2-3 steps also seems unfortuante to me. 

So this leaves us with 3 options:
a) Share a workflow, have "normal" processes printed to screen even if not run
b) Surround a module with `if` (although haven't figured out yet how to, as I don't like this solution)
c) Separate subworkflows

Reply:
Based on slack discussion: Use two separate workflows